Cargo.toml

```diff
@@ -24,4 +24,7 @@ decimal = "1"
 time = "0.1"
 uuid = "0.5"
 error-chain = "0.12.1"
-futures = "0.1"
```

Comment:
Please can you update the CHANGELOG? This will be a major version bump (to 0.15), since it introduces `std::futures`.

I also suspect some clients (including our internal one) will keep using the futures 0.1 version for a while, so we need to keep the 0.14.x stream around in case more changes are required there.

Reply:
Yup! Will take care of that. Wasn't sure if you wanted that before or after the merge :D